CONTRIBUTING.md

```diff
@@ -72,7 +72,10 @@ $ git commit -m "Update javadoc for API."
 Continuous integration builds the project, runs the tests, and runs multiple
 types of static analysis.
 
-1. Note: Currently, to run the full suite of tests, you'll need to be running a docker daemon.
+1. Note: Currently, to run the full suite of tests, you'll need to be running a docker daemon. If
+you cannot run the docker daemon, you can disable the docker tests by setting a gradle property of
+``"disable.docker.tests"`` to true. See the gradle.properties file in the root of the project
```

Comment:
I would do the opposite and disable by default, what do you think?

Reply:
I also support disabling it by default, and enabling it by default only for CircleCI.

Reply:
no strong opinion. My only concern is that we somehow get it disabled for CI and never notice.

Reply:
Personally, I like to always run them, so I'll probably just add the property to my personal gradle.properties and be fine.

Reply:
ok, reversed the logic and the name of the property (so we don't have double negatives everywhere).

Reply:
I think enabling by default but skipping the tests without error if container setup fails, unless explicitly enabled, would be best.

Reply:
So your suggestion is to always try, but only fail if the property is set to true?